draft-ietf-taps-arch.md

```diff
@@ -431,3 +389,7 @@ This work has been supported by the UK Engineering and Physical Sciences
 Research Council under grant EP/R04144X/1.
 
 Thanks to Stuart Cheshire, Josh Graessley, David Schinazi, and Eric Kinnear for their implementation and design efforts, including Happy Eyeballs, that heavily influenced this work.
+
+# Contributors
+
+The authors of this document include Tommy Pauly, Brian Trammell, Anna Brunstrom, Godred Fairhurst, Colin Perkins, Philipp S. Tiesel, and Chris Wood.
```

Comment:
I'd do this a bit differently (applies to all three docs)

- I'd place contributors above acknowledgments
- "include" sounds like there are other authors, which is not what we're saying.
- I haven't included editors in contributors' lists in the past "in addition to the editors, this document was written by:"
- I'd add back affiliations and (with author agreement, gdpr etc) email addresses
- and stylistically, I'd do this in list form.

Where specific sections have specific authors, this should also be noted (see 8095), but this is not the case here...

Reply:
Just to be sure ...

Are you suggesting keeping the current list of authors in the boilerplate; diving text between contributors and acknowledgment sections. - That I would agree with.

If you were to be suggesting pruning the authors list - I would not want that for the document s were I am currenty listed as an author: specifically I *would* wish to be in the RFC-Ed last call for these documents.

Reply:
See my answer to this in #184: I agree with Gorry here. Specifically, as long as we're not told that we MUST do such a thing, I think we should try to keep the author list as-is.